BasicGates/BasicGates.ipynb

```diff
@@ -546,7 +546,52 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Task 2.4. Toffoli gate\n",
+    "### Task 2.4. Two-qubit gate - 4\n",
+    "\n",
+    "**Input:** Two unentangled qubits (stored in an array of length 2) in an arbitrary two-qubit state ${\\color{blue}\\alpha} |00\\rangle + {\\color{blue}\\beta} |01\\rangle + \\gamma |10\\rangle + \\delta |11\\rangle$.\n",
+    "\n",
+    "\n",
+    "**Goal:**  Change the two-qubit state to ${\\color{red}\\beta} |00\\rangle + {\\color{red}\\alpha} |01\\rangle + \\gamma |10\\rangle + \\delta |11\\rangle$.\n",
+    "\n",
+    "> This task can be solved using one library function; as an exercise, try to express the solution using several (possibly controlled) Pauli gates.  \n",
+    ">\n",
+    "> If you can't come up with a solution, or if you have solved the task, read about the relevant library function: \n",
```

Comment:
Instead of this line, it can be reworded to a hint on lines of something like :
Interested in knowing about library function? Click here

Reply:
Thats a very good suggestion. I was to attached to the idea of teaching learners about this function 😅